IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcWellKnownTextCRS/DocEntity.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcWellKnownTextCRS" UniqueId="2c5451db-1148-4295-9d0e-fd2a3abfe638" BaseDefinition="IfcCoordinateReferenceSystem" EntityFlags="32">
```

Comment:
I'd rather see a specialized WKT atribute on each `IfcCoordinateReferenceSystem`.

Reply:
I agree with PJanck. An [OGC/ISO standard](https://www.ogc.org/standards/wkt-crs) compliant string containing the CRS information in a computer interpretable way would be of great help for geospatial software. 

All necessary scale adjustments (e.g. x/y-scale based on the height (h) above the ellipsoid) can be managed in the WKT. In terms of BIM quality assurance, each IFC data delivery can be checked to see whether it actually shows the agreed WKT string.

It is clear to me that it is not typical of IFC to write so many "chunks" of information as a single attribute in a string. However, that would be a good step in the right direction for the goal of automation-compatible BIM/GIS integration

Reply:
This has now been addressed with a new `IfcWellKnownText` entity.